src/sbd-inquisitor.c

```diff
@@ -1307,38 +1303,52 @@ int main(int argc, char **argv, char **envp)
         } else if (strcmp(argv[optind], "ping") == 0) {
             exit_status = ping_via_slots(argv[optind + 1], servants_leader);
 
-        } else if (strcmp(argv[optind], "watch") == 0) {
+        } else if (strcmp(argv[optind], "query-watchdog") == 0) {
```

Comment:
guess we need this to be outside the disk-only-section

Reply:
Yep those two should be separated as before.

Reply:
Guess we can have one long if-then-else-if-then ... clause with the first part disabled when built without disk-support.